changelog

```diff
@@ -1,3 +1,202 @@
+      * Mark Betelgeuse in "Deep Scientist Rescue 0" instead of making it a waypoint. (@roadrunner56)
+      * Use marks instead of waypoints in "Pact Recon 3". (@ziproot)
+    * Balance:
+      * Reduce the chances of finding Bactrians in random pirate fleet spawns by half. (@alexrovw)
+      * Nerf the Unfettered medical jobs. (@Hurleveur)
+      * Cut Wanderer sun reactor energy capacity in half + -20% energy generation on the biochemical cells. (@Quantumshark)
+      * Cut the asteroid scanner's price in half. (@AvianGeneticist)
+      * Reverted the changes to the acceleration and initial velocity of human missiles made in 0.10.0 (@Anarchist2)
+      * Tweaked the capacity of some human missile pods (@Anarchist2)
+        * Javelin mini pod 40 -> 30
+        * Sidewinder pod 4 -> 6
+        * Typhoon pod 2 -> 3
+      * Reduced the damage and increased the blast radius of the Heavy Rocket (and Pod). (@Quantumshark)
+        * "blast radius": 50 -> 90
+        * "shield damage": 1060 -> 790
+        * "hull damage": 900 -> 670
+      * Tweak Marauder loadouts, including removing campaign unlocked outfits. (@Hurleveur)
+      * Reduced the missile strength and turn rate of the Firelight. (@Quantumshark)
+        * "missile strength": 60 -> 30
+        * "Firelight Activated" "turn rate": 2 -> 1.7
+      * Give the Quicksilver +20 weapon capacity & a Capybara Reverse Thruster in exchange for its shield generator. (@Quantumshark)
+    * Other:
+      * Give the harvest personality to Unfettered fleets. (@Hurleveur)
+      * Edits to Unfettered missions. (@Hurleveur)
+      * Reworked the distribution of outfits in southern and pirate outfitters. (@bene_dictator)
+      * Remove facilities from evacuated Wanderer worlds. (@bene_dictator)
+      * Various improvements to human hails. (@bene_dictator)
+      * Update the descriptions of two Sestor worlds after the factories are shut down. (@Quantumshark)
+      * Add some additional mentions of how Tele'ek's Molt is progressing to the last few missions in Wanderers Middle. (@Quantumshark)
+      * Add mention of the CCOR in a FW pirate recon mission. (@roadrunner56)
+      * Removed pirates from Ildaria. (@Hurleveur)
+      * Adjust Imo Dep Minables & Add Hai Mining fleets. (@Saugia)
+      * Separate the Kestrel from "Tarazed Advanced" shipyard. (@Saugia)
+      * Gave Korath world-ships new display names based on planets in the original Korath territory. (@ziproot)
+  * Game mechanics:
+    * New mechanics:
+      * Autocondition for testing if you can land on a given planet. (@Amazinite)
+      * Added a "fused" weapon tag that causes projectiles to explode at the end of their life. (@Amazinite)
+      * Added tags that stop projectiles from colliding with different object types. (@Amazinite)
+      * Add a "quiet" personality that prevents passive hails but still allows direct hails. (@bene_dictator)
+      * Conditions to word-form numbers in conversations. (@UnorderedSigh)
```

Comment:
Maybe something about adding syntax for that?

Reply:
```suggestion
      * Added support for substituting the values of conditions in word form within conversation text. (@UnorderedSigh)
```

Reply:
The first one is probably better. You might have to change some I didn't comment on, too.